cpp/tensorrt_llm/batch_manager/trtGptModelInflightBatching.cpp

```diff
@@ -2601,4 +2615,169 @@ SizeType32 TrtGptModelInflightBatching::getMaxCapacityBatchSize(SizeType32 input
     return mKvCacheManager->getMaxCapacityBatchSize(inputLength, outputLength);
 }
 
+void TrtGptModelInflightBatching::prefetchPromptTableChunk(
```

Comment:
will do more careful review in 2nd round, but firstly can one of prefetchPromptTableChunk and prefetchPromptTableChunkToGpu be renamed to be less confusing? right now they're overlapping in naming